ui/app/pages/swaps/swaps.util.js

```diff
@@ -323,6 +327,17 @@ export async function fetchSwapsFeatureLiveness() {
   return status?.active
 }
 
+export async function fetchSwapsQuoteRefreshTime() {
+  const response = await fetchWithCache(
+    getBaseApi('refreshTime'),
+    { method: 'GET' },
+    { cacheRefreshTime: 600000 },
+  )
+
+  // We presently use milliseconds in the UI
+  return response?.seconds * 1000
```

Comment:
If the `seconds` property is not present, this will return `NaN`. Perhaps it should throw an error in this case instead? I don't think `NaN` would be a particularly helpful response.

Reply:
🙌